packages/web-app-files/src/composables/actions/files/useFileActionsSetImage.ts

```diff
@@ -34,7 +35,6 @@ export const useFileActionsSetImage = ({ store }: { store?: Store<any> } = {}) =
       const { data } = await graphClient.drives.updateDrive(
         storageId,
         {
-          name: space.name,
```

Comment:
We were discussing whether we wanted to implement the Drive interface fully or just cast our data and we assumed that setting more data than we actually want to change might trigger unexpected behavior for users, in this case e.g. reverting a name change done by another user by uploading a new space image